File: components/policy/resources/webui/policy.js
FASTVC-BENCH-CORPUS:chromium-main-v1-943b94ae-1c74-4335-94fa-ceb4d277cea8
// Copyright (c) 2013 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

import {Page} from './policy_base.js';

// Have the main initialization function be called when the page finishes
// loading.
const page = Page.getInstance();
document.addEventListener('DOMContentLoaded', () => {
  page.initialize();
});

File: sandbox/linux/bpf_dsl/policy.cc
FASTVC-BENCH-CORPUS:chromium-main-v1-943b94ae-1c74-4335-94fa-ceb4d277cea8
// Copyright 2014 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

#include "sandbox/linux/bpf_dsl/policy.h"

#include <errno.h>

#include "sandbox/linux/bpf_dsl/bpf_dsl.h"

namespace sandbox {
namespace bpf_dsl {

ResultExpr Policy::InvalidSyscall() const {
  return Error(ENOSYS);
}

}  // namespace bpf_dsl
}  // namespace sandbox
